doc-content/enterprise-only/optimizer/quarkus-creating-proj-proc.adoc

```diff
@@ -0,0 +1,93 @@
+[id='proc-quarkus-creating-proj_{context}']
+
+
+= Creating the school timetable project
+
+The school timetable project lets you get up and running with {PLANNER} and Quarkus application using Apache Maven and the Quarkus Maven plug-in.
+
+.Procedure
+
+. In a command terminal, enter the following command to verify that Maven is using JDK 11 and that the Maven version is 3.6 or higher:
++
+[source]
+----
+mvn --version
+----
+
+. If the preceding command does not return JDK {JDKVersion}, add the path to JDK {JDKVersion} to the PATH environment variable and enter the preceding command again.
+
+. To generate the project, enter one of the following commands:
++
+NOTE: Apple macOS and Microsoft Windows are not supported production environments.
++
+* If you are using Linux or Apple macOS, enter the following command:
++
+[source,shell,subs=attributes+]
+----
+mvn io.quarkus:quarkus-maven-plugin:1.11.6.Final:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=optaplanner-quickstart \
```

Comment:
``` 
mvn io.quarkus:quarkus-maven-plugin:1.11.6.Final-redhat-00001:create \
    -DprojectGroupId=org.acme \
    -DprojectArtifactId=optaplanner-quickstart  \
    -Dextensions="resteasy,resteasy-jackson,optaplanner-quarkus,optaplanner-quarkus-jackson"    \
    -DplatformGroupId=com.redhat.quarkus \
    -DplatformVersion=1.11.6.Final-redhat-00001 \
    -DnoExamples\
```

This is the right command - it creates an no example project with OptaPlanner dependencies

RESTEasy JAX-RS (quarkus-resteasy)
RESTEasy Jackson (quarkus-resteasy-jackson)
OptaPlanner (optaplanner-quarkus)
OptaPlanner Jackson (optaplanner-quarkus-jackson)

And substitutes right quarkus-universe version

Reply:
Feel free to check it here https://docs.google.com/document/d/1tmYhkpciXJW6ut6vDwBOD5ZxCZmBXHrxm5JEl4UB49E/edit#

Reply:
Thanks for the property explanation table, 
maybe we can complete the section with 

"extensions | A comma-separated list of extensions to use with this project."
With stress on Quarkus extensions? The full list of the extensions could be found 
`mvn quarkus:list-extensions`
or https://quarkus.pro/extensions/

Reply:
Could you please remove the "/" at the end of the mvn command. (in the end of     -DnoExamples\)

Reply:
Both done. I didn't link to the doc. We aren't supposed to link to community docs and it doesn't look like Red Hat build of Quarkus has migrated that content yet.

Reply:
Fixed